src/plugins/discover/public/application/components/sidebar/discover_sidebar.tsx

```diff
@@ -231,11 +231,12 @@ export function DiscoverSidebar(props: DiscoverSidebarProps) {
               />
             </EuiSplitPanel.Inner>
           ) : null}
+
           <EuiSplitPanel.Inner
             className="eui-yScroll dscSideBar_fieldListContainer"
             paddingSize="none"
           >
-            {fields.length > 0 && (
+            {(fields.length > 0 || selectedIndexPattern.fieldsLoading) && (
```

Comment:
if loading and `fields` list is empty, what does UI show? should it indicate that it's loading?

Reply:
If loading and fields list is empty -> The UI shows the sidebar in the loading state.